src/cio.ts

```diff
@@ -1,18 +1,15 @@
 import { CustomerIORequestError, TrackClient } from 'customerio-node';
 import { ChangeObject } from './types';
 import {
+  ConnectionManager,
   User,
   UserPersonalizedDigest,
   UserPersonalizedDigestType,
   UserStreak,
 } from './entity';
-import {
-  camelCaseToSnakeCase,
-  CioUnsubscribeTopic,
-  debeziumTimeToDate,
-  getFirstName,
-  getShortGenericInviteLink,
-} from './common';
+import { camelCaseToSnakeCase, debeziumTimeToDate } from './common/utils';
+import { CioUnsubscribeTopic, getFirstName } from './common/mailing';
+import { getShortGenericInviteLink } from './common/links';
 import type { UserCompany } from './entity/UserCompany';
```

Comment:
I'm getting some weird issues that the imports are not defined. Had to separate each to make them work.